test/integration/ssl_integration_test.cc

```diff
@@ -23,16 +25,16 @@ void SslIntegrationTest::SetUp() {
   context_manager_.reset(new ContextManagerImpl(*runtime_));
   upstream_ssl_ctx_ = createUpstreamSslContext();
   fake_upstreams_.emplace_back(
-      new FakeUpstream(upstream_ssl_ctx_.get(), 0, FakeHttpConnection::Type::HTTP1));
+      new FakeUpstream(upstream_ssl_ctx_.get(), 0, FakeHttpConnection::Type::HTTP1, GetParam()));
   registerPort("upstream_0", fake_upstreams_.back()->localAddress()->ip()->port());
```

Comment:
I continue to be an unfan of the use of `GetParam()` in so many places. It's easy to lose track of what it's referring to. I would prefer a `GetIpVersion()` or something similar.